metpy/calc/thermo.py

```diff
@@ -855,3 +855,116 @@ def relative_humidity_from_specific_humidity(specific_humidity, temperature, pre
     return (100 * units.percent *
             mixing_ratio_from_specific_humidity(specific_humidity) /
             saturation_mixing_ratio(pressure, temperature))
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def cape_cin(pressure, temperature, dewpt, parcel_profile):
+    r"""Calculate CAPE and CIN.
+
+    Calculate the convective available potential energy (CAPE) and convective inhibition (CIN)
+    of a given upper air profile and parcel path. CIN is integrated between the surface and
+    LFC, CAPE is integrated between the LFC and EL (or top of sounding). Intersection points of
+    the measured temperature profile and parcel profile are linearly interpolated.
+
+    Parameters
+    ----------
+    pressure : `pint.Quantity`
+        The atmospheric pressure level(s) of interest. The first entry should be the starting
+        point pressure.
+    temperature : `pint.Quantity`
+        The starting temperature
+    dewpt : `pint.Quantity`
+        The starting dew point
+    parcel_profile : `pint.Quantity`
+        The temperature profile of the parcel
+    Returns
+    -------
+    `pint.Quantity`
+        Convective available potential energy (CAPE).
+    `pint.Quantity`
+        Convective inhibition (CIN).
+    See Also
+    --------
+    lfc, el
+
+    """
+    # Calculate limits of integration
+    lfc_pressure = lfc(pressure, temperature, dewpt)[0]
```

Comment:
It seems wrong to have LFC/EL not depend at all on the parcel profile...am I missing something?